src/esp/physics/ArticulatedObject.h

```diff
@@ -46,6 +47,94 @@ enum class JointType : int {
   Invalid
 };
 
+////////////////////////////////////
+// Joint Motor Interface
+////////////////////////////////////
+
+//! describes the type of a motor for generality.
+enum class JointMotorType { SingleDof, Spherical };
+
+/**
+ * @brief Stores JointMotor (i.e. per-DoF PD control) parameters for creation
+ * and updates.
+ */
+struct JointMotorSettings {
+ public:
+  JointMotorSettings() = default;
+
+  //! constructor for single dof motor settings
+  JointMotorSettings(double _positionTarget,
+                     double _positionGain,
+                     double _velocityTarget,
+                     double _velocityGain,
+                     double _maxImpulse) {
+    positionTarget = _positionTarget;
+    positionGain = _positionGain;
+    velocityTarget = _velocityTarget;
+    velocityGain = _velocityGain;
+    maxImpulse = _maxImpulse;
+  }
+
+  //! constructor for spherical motor settings
```

Comment:
Do docs with this format get picked up in our documentation?  If not I think it would be worth the effort to refactor them to appropriate docs.

Reply:
Yeah, these get picked up. A bit lazy, but fine IMO for small doc briefs.